### ISSUE TITLE
Compact track buttons leads to crash

### ISSUE THREAD

**Issue by BaraMGB** [LMMS/lmms#3454]
From discord (@qnebra reported) 

>There you are my little friend, enabling "compact track buttons" cause massive segfault on me.

 I can reproduce this.

**Comment by zonkmachine** [comment]
> enabling "compact track buttons" cause massive segfault on me.

I don't see this. `Version 1.2.0-rc2.107 (Linux/x86_64, Qt 4.8.6, GCC 4.8.5)`
Can you give the steps to reproduce this. Is it just clicking the checkbox in Settings?

**Comment by softrabbit** [comment]
> I don't see this. Version 1.2.0-rc2.107 (Linux/x86_64, Qt 4.8.6, GCC 4.8.5)
Can you give the steps to reproduce this. Is it just clicking the checkbox in Settings?

Enable it, restart LMMS, click on the synth logo for a track in the song editor. Segfault.
LMMS 1.2.0-rc2.108 (Linux x86_64, Qt 5.5.1, GCC 5.4.0 20160609)

**Comment by Lukas-W** [comment]
This is the commit that introduced this crash: 91182b4adcb9995240d1e82e157ac8131a957125. When "compact track buttons" is enabled, `m_renameLineEdit` is null:
```cpp
void TrackLabelButton::mouseReleaseEvent( QMouseEvent *_me )
{
	if( m_buttonRect.contains( _me->globalPos(), true ) && m_renameLineEdit->isHidden() )
//                                                                      ^
	{
		QToolButton::mousePressEvent( _me );
	}
	QToolButton::mouseReleaseEvent( _me );
	_me->ignore();
}
```

**Comment by BaraMGB** [comment]
Yes, I guessed something like that. Therefore I assigned myself for this. :D

**Comment by qnebra** [comment]
Thanks @BaraMGB for raport this.

**Comment by tresf** [comment]
Closed via #3463
